core/templates/components/question-difficulty-selector/question-difficulty-selector.directive.html

```diff
@@ -14,3 +16,16 @@
     </md-list>
   </md-radio-button>
 </md-radio-group>
+
+<style type="text/css">
```

Comment:
Do we specify `text/css` anywhere else? Is it required?

Reply:
No, it's not required. I've removed it now.